packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -0,0 +1,1073 @@
+//! Implementation of formatting, to enable colors and the use of box-drawing characters use the
```

Comment:
Do I see correctly, that the whole `fmt` module is only available with `feature = "std"`? We might want to feature-flag it as well.

Reply:
No, the whole `fmt` module is enabled in both `std` and `no-std`. Actually you can even have colors and such with `no-std`!

Reply:
The only difference is that `std` enables `Hooks`, which `no-std` doesn't support, but the output (`Debug`, `Display`, built-in, and the tree) is supported.

Reply:
Hmm, what about making it only `pub` on `feature = "std"`? (See below)

Reply:
Sounds good. We only export items on `std` anyway (except for `Emit`), so this would be better.